app/views/alaveteli_pro/info_request_batches/_message_preview.html.erb

```diff
@@ -27,7 +27,18 @@
               'request to make a new but similar batch of requests.',
               date: simple_date(@info_request_batch.existing_batch.created_at),
               existing_batch: info_request_batch_path(@info_request_batch.existing_batch)) %>
+
+        <%= _('As a Pro user, you can choose to ignore the existing batch by ' \
+              'checking the box below and send your requests. Please ' \
+              'remember you are still bound by our <a href="{{url}}">House ' \
+              'Rules on making responsible requests</a>.',
+              url: help_requesting_path(anchor: 'responsible')) %>
       </div>
+
+      <%= label_tag :ignore_existing_batch do %>
+        <%= check_box_tag :ignore_existing_batch %>
+        <%= _('Ignore existing batch') %>
```

Comment:
- [ ] Should disable the submit tag unless this is checked.
- [ ] Change wording to "Send this batch anyway?"